src/design-system/components/Text/Text.docs.tsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```

Comment:
hm, maybe we can define just other rules for this package?

Reply:
Normally I don't mind this rule, it's just awkward in this case because it's moving the `name` property *below* the `Example` property which makes it annoying to read, especially with longer examples. Maybe I'll just get creative and come up with property names that can be sort alphabetically 😅